3.11/release-notes-new-features311.md

```diff
@@ -233,3 +233,19 @@ You can configure the feature via the following new startup options:
   Note that an auto-flush is only executed if the number of live WAL files
   exceeds the configured threshold and the last auto-flush is longer ago than
   the configured auto-flush check interval. This avoids too frequent auto-flushes.
+
+## Miscellaneous changes
+
+### Trace logs for graph traversals
+
+Detailed information is now logged if you run AQL graph traversals and set the
+log level to `TRACE` for the `graphs` log topic. This information is fairly
+low-level but can help to understand correctness and performance issues with
+traversal queries. There are also some new log messages for the `DEBUG` level.
+
+To enable tracing for traversals at startup, you can set `--log.level graphs=trace`.
```

Comment:
`Traversals` is valid for all graph searches we have? Because this statement is valid for all graph searches we have: 

![image](https://user-images.githubusercontent.com/1244163/231117408-6ce076d7-9596-4dca-b2f0-9fb22328c909.png)